Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/Utilities/VRTK_ContentHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace VRTK
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// This script will make objects inside of control contents properly interactable by reparenting them. 
+    /// Otherwise they would disappear if e.g. a drawer is closed even if the object was outside the drawer.
+    /// We need to go through collisions instead of hooking to the Ungrabbed event since ungrabbing will most likely happen above the inside causing the object to drop first. 
+    /// This approach is much more stable and allows for many more scenarios.
+    /// Not supported yet: detect if an object is taken from one content manager and dropped into another one.
+    /// </summary>
+    public class VRTK_ContentHandler : MonoBehaviour
+    {
+        public VRTK_Control control;
+        public Transform inside;
+        public Transform outside;
+
+        void Start()
```

Comment:
all methods/variables should specify public/protected/private as per the contrib guidelines.

Otherwise there is a mix and match of when it's included and when it's not.